dotnet/src/AutoGen.SourceGenerator/Template/FunctionCallTemplate.tt

```diff
@@ -81,7 +81,7 @@ namespace <#=NameSpace#>
                         Name = @"<#=parameter.Name#>",
 <#}#>
 <#if (parameter.Description != null) {#>
-                        Description = @"<#=parameter.Description#>",
+                        Description = @"<#= parameter.Description.Replace("\"", "\"\"") #>",
 <#}#>
 <#if (parameter.Type != null) {#>
                         ParameterType = typeof(<#=parameter.Type#>),
```

Comment:
Can you also update the parameter's description (double `"`) as well?

Reply:
Done!